Update PR Build Action for API (#3535)
GeyserMC/Geyser: CONTRIBUTING.md

The nature of our software requires a lot of arrays and maps to be stored - where possible, use Fastutil's specialized maps. For example, if you're storing block state translations, use an `Int2IntMap`.

We have a rundown of all the tools you need to develop over on our [wiki](https://wiki.geysermc.org/other/developer-guide/). If you have any questions, please feel free to reach out to our [Discord](https://discord.gg/geysermc)!

> If making a pull request that also depends on changes to [Geyser's API](https://github.com/GeyserMC/api), simply fork the API repo and create a branch with the same name as your PR to Geyser. The pull request [action](https://github.com/GeyserMC/Geyser/blob/master/.github/workflows/pullrequest.yml) will automatically cache your Geyser API changes before building your changes to Geyser.

Reviewer: ```suggestion
If you're making a pull request that also depends on changes to [the base API](https://github.com/GeyserMC/api), simply fork the API repo and create a branch with the same name as your Geyser PR. The pull request [action](https://github.com/GeyserMC/Geyser/blob/master/.github/workflows/pullrequest.yml) will automatically use your API changes while building your changes to Geyser.
```

Author: https://github.com/GeyserMC/Geyser/pull/3535/commits/5ebfc0f361518d72b5c69c9f0b46899ff9b3e9ae#diff-ce9a2f9036cc2a5aaf1f78839c4f65c7a12808822e0dd0f3552471921ac5e007L25-R25
